apps/include/apps.h

```diff
@@ -40,6 +40,14 @@
 #  define openssl_fdset(a,b) FD_SET(a, b)
 # endif
 
+#if defined(OPENSSL_SYS_WINDOWS) || defined(OPENSSL_SYS_MSDOS)
+# include <conio.h>
+#endif
+
+#if defined(OPENSSL_SYS_MSDOS) && !defined(_WIN32)
+# define _kbhit kbhit
+#endif
```

Comment:
These should be moved to `apps/apps.c`, since that's the only consumer.  (they are used to implement `has_stdin_waiting`)

Reply:
Done (master and 1.1.1)